pandas/tests/resample/test_datetime_index.py

```diff
@@ -124,7 +124,7 @@ def test_resample_integerarray():
     expected = Series(
         [1, 4, 7],
         index=pd.date_range("1/1/2000", periods=3, freq="3T"),
-        dtype="float64",
+        dtype="Int64",
```

Comment:
I think it's okay that this is now returning Int64?

Reply:
I think this is OK as this is reverting the test to before #31359 which is the cause of the regression in #32194. There is a whatsnew in #31359 about a change in behavior. can you double check that this change is ok in that context.